### PR TITLE
Add tracking for dismissing splash page

### PR DESCRIPTION
**Pull request by alanmoo**
A little bit of #446

### CODE REVIEW

File: source/js/components/takeover/takeover.jsx [hunk 1]
```diff
@@ -8,6 +9,12 @@ export default class Takeover extends React.Component {
       isHidden: true
     });
 
+    ReactGA.event({
+      category:`splash`,
+      action: `splash exit`,
+      label: `dismiss splash `
```
Comment (xmatthewx) [inline source/js/components/takeover/takeover.jsx]: For label, can distinguish between the two buttons? Using button string or add a name to element?

**Comment by cadecairos** [comment]
Does react-ga just plug into existing GA scripts on the page? I though it needed to be i[nitialized before use](https://www.npmjs.com/package/react-ga#reactgainitializegatrackingid-options).

**Comment by alanmoo** [comment]
When I tested this it seemed to fire the event upon dismissing the splash modal, so I thought it just hooked right in.

**Comment by alanmoo** [comment]
If this needs changes (I'm guessing the `logJoinBtn` function isn't going to get called, but I can't confirm) can someone take this and run with it so we can get this in today?

**Comment by alanmoo** [comment]
Even if we use the anchor link tracking, we should make sure we can identify a user as coming from the button on the splash vs the button in the header

File: package.json [hunk 2]
```diff
@@ -80,6 +80,7 @@
     "mofo-bootstrap": "3.2.0",
     "react": "15.4.2",
     "react-dom": "15.4.2",
+    "react-ga": "^2.2.0",
```
Comment (gvn) [inline package.json]: Lock to `2.2.0`

**Pull request merged**